pkg/api/api_test.go

```diff
@@ -46,6 +48,58 @@ func testSetup() (*testEnv, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// Constructors for ServiceDisruptor and PodDisruptor will error if they cannot find any target for the supplied
+	// parameters. For this reason, we need to add to the fake k8s client a service and a pod backing it.
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{Name: "namespace"},
```

Comment:
Why is a nondefault namespace required?

Reply:
I'm not sure it is required, but it needs to match what tests are expecting to find:

https://github.com/grafana/xk6-disruptor/blob/468f42766604b5790066b64dadf113f662b29696/pkg/api/api_test.go#L121-L128

Reply:
Good catch, I guess that previously didn't make a difference because the constructor wasn't checking for a match.